README.md

```diff
@@ -25,7 +25,9 @@ states and produce helpful logs.
 
 Install and run Matrix by doing the following:
 
-    sudo pip3 install 'git+https://github.com/juju-solutions/matrix.git'
+    git clone https://github.com/juju-solutions/matrix.git
+    cd matrix
+    sudo pip3 install . -f wheelhouse
```

Comment:
I don't like telling people to install in their system Python. :-/

This command (minus the sudo) fails in a virtualenv, as it can't find zookeeper.h (probably a lib dir missing from the include path).